sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -33,41 +32,23 @@
 @RestController
 @RequestMapping("/orchestration")
 class OrchestrationController {
-
-  private static final OrchestrationClient CLIENT = new OrchestrationClient();
-
   static final String MODEL = "gpt-35-turbo";
 
   private static final LLMModuleConfig LLM_CONFIG =
       LLMModuleConfig.create().modelName(MODEL).modelParams(Map.of());
+  private static final ModuleConfigs config =
+      ModuleConfigs.create()
+          .llmModuleConfig(LLM_CONFIG)
+          .templatingModuleConfig(TemplatingModuleConfig.create().template());
```

Comment:
Note that this config object is reused for multiple requests. See the [usage here](https://github.com/SAP/ai-sdk-java/pull/140/files#diff-f4eba075ea74372409972dd8efc07ded54bd91294a660f2bacf05e5f7f40cc69R174) and the [implementation here](https://github.com/SAP/ai-sdk-java/pull/140/files#diff-53dc72dc76c635ec0e6057e9cc9bacaef0034034b5b8e96dfd1af46e3fb64f94R18) for why this works.

However, this is quite fragile, since there is no deep copy. If users manipulate objects at the wrong level they will accidentally manipulate the shared config object.